cognite/client/data_classes/aggregates.py

```diff
@@ -0,0 +1,10 @@
+from typing import *
+
+from cognite.client.data_classes._base import CognitePropertyClassUtil
+
+
+class Aggregate(dict):
+    def __init__(self, count: int = None):
+        self.count = count
```

Comment:
there a few things I don't like about this file:

First of all it assumes at aggregated responses will always have consistent format across all resource types. Which is not always the case if we look retrospectively

Secondly, this class is not a resource type, so it probably shouldn't have its own file. Should be more like be a part of something else, like `shared.py`? (this might not make sense after you look at the next point)

And lastly we need to generate as much as we can, so I suggest to use the generator, (look how `# GenPropertyClass: AggregateResultItem` is generated)
Then we'll have it's own aggregation response in every resource type

You can have a look for schema names here https://storage.googleapis.com/cognitedata-api-docs/dist/v1.json
for assets it's called "AssetAggregate" for instance.
They have inconsistent names across resource type so you might need to rename some (generator can do that)

Reply:
Fixed